libs/utils/analysis/tasks_analysis.py

```diff
@@ -661,6 +663,14 @@ def _plotTaskSignals(self, axes, tid, signals, is_last=False):
             axes.axhline(tip_bcap, color='r', linestyle=':', linewidth=2)
             axes.axhline(max_bcap, color='r', linestyle='--', linewidth=2)
 
+        # Add any other additional optimal specified signal
+        for signal in signals:
+            if signal in ['load_avg', 'util_avg', 'boosted_util']:
+                continue
+            if signal not in util_df.columns:
+                continue
+            data = util_df[util_df.pid == tid][signal]
+            data.plot(ax=axes, drawstyle='steps-post', legend=True)
```

Comment:
I'm confused here why, don't we combine this with the but that plots `load_avg` and `util_avg` at the top of the function? Can't we just change the computation of `signals_to_plot` to something like 

    signals_to_plot = set(signals).difference({'boosted_util'})

?